netbox/extras/views.py

```diff
@@ -1479,6 +1482,34 @@ def get(self, request, **kwargs):
         if job.completed:
             table = self.get_table(job, request, bulk_actions=False)
 
+        if job.completed and job.data:
```

Comment:
The export function should still work even if the data is empty. Otherwise, the user will not be able to determine if the request is being processed as intended.